staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4981,6 +4981,18 @@ type ServiceProxyOptions struct {
 }
 
 // ObjectReference contains enough information to let you inspect or modify the referred object.
```

Comment:
is there a way to limit this text to godoc and prevent it from appearing in the openapi description?